apps/src/templates/studioHomepages/SeeMoreCourses.jsx

```diff
@@ -35,13 +44,15 @@ const SeeMoreCourses = React.createClass({
             isRtl={isRtl}
           >
             {courses.map((course, index) =>
-              <CourseCard
-                key={index}
-                name={course.name}
-                description={course.description}
-                link={course.link}
-                isRtl={isRtl}
-              />
+              <div key={index}>
+                <CourseCard
+                  name={course.name}
+                  description={course.description}
+                  link={course.link}
+                  isRtl={isRtl}
+                />
+                {(index % 2 === 0) && <div style={styles.spacer}>.</div>}
```

Comment:
Is this spacer going to be a common thing used throughout our code? I wonder if we should just have a Spacer component?

Would also be nice to include a screenshot.